File: eform-client/src/app/models/cases/response/case-list.model.ts
import {CaseModel} from 'app/models';

export class CaseListModel {
  numOfElements: number;
  pageNum: number;
  cases: Array<CaseModel> = [];

  constructor() {
    this.pageNum = 0;
    this.numOfElements = 0;
  }
}

File: old.eFromAPI/eFormAPI/Infrastructure/Models/Cases/Response/CaseListModel.cs
using System.Collections.Generic;

namespace eFormAPI.Web.Infrastructure.Models.Cases.Response
{
    public class CaseListModel
    {
        public int NumOfElements { get; set; }
        public int PageNum { get; set; }
        public List<Case> Cases { get; set; }
    }
}